Makefile

```diff
@@ -163,16 +170,54 @@ docker-push: ## Push docker image with the manager.
 # - have enabled BuildKit. More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 # - be able to push the image to your registry (i.e. if you do not set a valid value via IMG=<myregistry/image:<tag>> then the export will fail)
 # To adequately provide solutions that are compatible with multiple platforms, you should consider using this option.
-PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
+PLATFORMS ?= linux/arm64,linux/amd64
+BUILDER_NAME ?= inferno-autoscaler-builder
+
+.PHONY: docker-buildx-setup
+docker-buildx-setup: ## Setup docker buildx builder for multi-arch builds
+	@echo "Setting up docker buildx builder..."
+	@$(CONTAINER_TOOL) buildx inspect $(BUILDER_NAME) >/dev/null 2>&1 || \
+		$(CONTAINER_TOOL) buildx create --name $(BUILDER_NAME) --driver docker-container --use
+	@$(CONTAINER_TOOL) buildx inspect --bootstrap
+
 .PHONY: docker-buildx
-docker-buildx: ## Build and push docker image for the manager for cross-platform support
-	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
-	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
-	- $(CONTAINER_TOOL) buildx create --name inferno-autoscaler-builder
-	$(CONTAINER_TOOL) buildx use inferno-autoscaler-builder
-	- $(CONTAINER_TOOL) buildx build --push --platform=$(PLATFORMS) --tag ${IMG} -f Dockerfile.cross .
-	- $(CONTAINER_TOOL) buildx rm inferno-autoscaler-builder
-	rm Dockerfile.cross
+docker-buildx: docker-buildx-setup ## Build and push docker image for the manager for cross-platform support
```

Comment:
Why did we add new buildx commands? They are working on MAC and x86

Reply:
this is removed